core/src/lib.rs

```diff
@@ -1,6 +1,13 @@
 #![deny(warnings)]
+
+#[macro_use]
+extern crate serde_derive;
+extern crate serde;
+extern crate serde_json;
+
 extern crate holochain_dna;
 extern crate wabt;
```

Comment:
I would order the import this order:
```rust
extern crate foo;
extern crate bar;

mod blah;
mod blahblah;
mod blahblahblah;

use foo::Bar;
use foo::Bang;
use foo::Boom;
use foo::Bang;
```
**But** we have *external* crates and *internal* crates, the ones that are in the repo. SO I'm thinking of having the external ones above the internal one with a gap in between.

Reply:
what are you wanting me to change here?

Reply:
For example, `holochain_dna` is *internal*, so I would move it below the *external* crates with an empty line between